.github/workflows/build.yml

```diff
@@ -3,12 +3,27 @@
 name: Build
 
 on:
-  workflow_dispatch
-
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Wazuh version'
+        required: true
+        default: '4.4.0'
+      revision:
+        description: 'Wazuh revision'
+        required: true
+        default: '1'
+      wazuhappurl:
+        description: 'Wazuh app zip URL'
+        required: true
+        default: 'https://packages-dev.wazuh.com/pre-release/ui/dashboard/wazuh-4.4.0-1.zip'
```

Comment:
Would it be possible to use `inputs.version` by default? 

Also, this input could be optional, using `https://packages-dev.wazuh.com/pre-release/ui/dashboard/wazuh-${{ github.event.inputs.version }}-1.zip` as URL for the plugin.

Reply:
It should be possible, will work on it

Reply:
Will discuss about it to possibly add it in a future PR